api/models/settings.py

```diff
@@ -0,0 +1,237 @@
+from api.db import db
+
+
+class ApplicationEnvironament:
+
+    def __init__(self):
+        pass
+    DEVELOPMENT = 'development'
+    STAGING = 'staging'
+    PRODUCTION = 'production'
+    TESTING = 'testing'
+
+
+class Settings(db.Model):
+    __tablename__ = 'settings'
+
+    app_environment = db.Column(db.String, default=ApplicationEnvironament.PRODUCTION)
```

Comment:
It won't be a radio button as the user can send any malicious string to the backend. Validators should validate the input from the given set of config.